servicex/templates/did-finder-demo/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Could/should this be wrapped in a conditional so it only is deployed if requested in `values.yaml`?